pkg/db/drivers/etcd/etcd.go

```diff
@@ -578,9 +578,7 @@ func (c *Client) UpdateFileShare(ctx *c.Context, fshare *model.FileShareSpec) (*
 	if fshare.Name != "" {
 		result.Name = fshare.Name
 	}
-	if fshare.Description != "" {
-		result.Description = fshare.Description
-	}
+	result.Description = fshare.Description
```

Comment:
It's not allowed to update empty description.

Reply:
+1

Reply:
+1